app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, first, second = command.split(" ")
```

Comment:
give more descriptive names for variables, also `split()` already has a default value `" "`  no need to write it